script.js

```diff
@@ -0,0 +1,50 @@
+import encyptionModule from './modules/encryption';
+import decryptionModule from './modules/decryption';
+import './style.css';
+
+function onClick(operationType) {
+	const inputVal = document.querySelector('.inputstring').value;
+	if (operationType === 'encrypt') {
+		if (!/^[a-zA-Z]+$/.test(inputVal)) {
+			alert('Please input just characters');
+			return;
```

Comment:
Cool handling. Move to separate function please

Reply:
ok